ga4gh/backend.py

```diff
@@ -570,6 +580,15 @@ def callSetsGenerator(self, request):
             request, variantSet.getCallSetIdMap(),
             variantSet.getCallSetIds())
 
+    def genotypePhenotypeGenerator(self, request):
+        """
+        Returns a generator over the (genotypePhenotype, nextPageToken)
+        pairs defined by the specified request.
+        """
+        intervalIterator = GeneotypePhenotypeIterator(
```

Comment:
We should do more work in here, and remove the GeneotypePhenotypeIterator. All you really need is to get the offset from the pageToken, and then pass this on to the query method in the data model object.